README.md

```diff
@@ -109,6 +109,8 @@ audio = "ask" # "yes", "no"
 [notification_actions]
 # Alternative name for dragon-drop binary
 dragon.command = "dragon"
+# Alternative name for swappy command : {} will be replaced by the file path to the screenshot
+swappy.command = "swappy -f {} -o {}"
```

Comment:
```suggestion
edit.command = "swappy -f {} -o {}"
```

Otherwise it doesn't work as the option is called `edit` and not `swappy` down the line.